third_party/terraform/website/docs/r/compute_region_instance_group_manager.html.markdown

```diff
@@ -138,6 +143,8 @@ group. You can specify only one value. Structure is documented below. For more i
 * `distribution_policy_zones` - (Optional) The distribution policy for this managed instance
 group. You can specify one or more values. For more information, see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/distributing-instances-with-regional-instance-groups#selectingzones).
 
+* `distribution_policy_target_shape` - (Optional) The shape to which the group converges either proactively or on resize events (depending on the value set in update_policy.0.instance_redistribution_type).
```

Comment:
Are there a set of possible values that this can be? Maybe include either a link to documentation about them or a list of possible values, as it's difficult to tell what these could be

Reply:
I didn't add them in a end users are expected to read them from the API docs or the Google docs. Also the values can change and I have been seeing a number of issues recently about stale docs which we can avoid by validating fields or noting values in TF docs.

Reply:
Can we explicitly link to the docs from this description? Something like the "For more information, see the official documentation..." in the block above.

This page could work, as it lists the possible types: https://cloud.google.com/compute/docs/instance-groups/regional-mig-distribution-shape

Reply:
OK.

Reply:
fixed